projects/BFDR.Messaging/BFDRAcknowledgementMessage.cs

```diff
@@ -11,7 +12,7 @@ public class BirthRecordAcknowledgementMessage : BFDRAcknowledgementMessage
         public new const string MESSAGE_TYPE = "http://nchs.cdc.gov/birth_acknowledgement";
 
         /// <summary>Default constructor that creates a new, empty BirthRecordAcknowledgementMessage.</summary>
-        public BirthRecordAcknowledgementMessage(BFDRBaseMessage message) : base(message)
+        public BirthRecordAcknowledgementMessage(CommonMessage message) : base(message)
```

Comment:
We might not want this to take CommonMessage since we may only want BFDRBaseMessages to be passable here. It makes things easier to implement in the API this way, but I can work around it if need be.

Reply:
I've been debating this one. It seems like the main advantage to this was it cut down the amount of code needed in the API's auto-acknowledgement code we use for testing. Is that accurate? Would the alternative to this approach mean we would just have more code in the API or is there other functionality it enables?

Reply:
Yes, It mostly cuts down on the amount of code needed in places like the API, but only a trivial amount since it would need to just be casted anyway. But at the same time, switching back to BFDRBaseMessage doesn't really make much of a difference to having CommonMessage, right? Because BFDRBaseMessage is not Birth-specific anyway, and it gets passed in the same way for FetalDeathAcknowledgementMessage() as well.

Reply:
That's true, in this case it doesn't really add value to use BFDRBaseMessage. I'm good with this change.